app/templates/submit.hbs

```diff
@@ -156,6 +166,18 @@
                                     </div>
                                 </div>
                                 <div class="row">
+                                    <div class="col-md-6">
+                                        <h4>{{if newNode 'Apply' 'Also apply'}} this license to my OSF Project.</h4>
```

Comment:
Nevermind, I'm not sure if I was looking at something different or the spec has changed slightly, but you are right!